src/payment/strategies/ppsdk/ppsdk-strategy.spec.ts

```diff
@@ -121,24 +122,6 @@ describe('PPSDKStrategy', () => {
                     });
                 });
             });
-            describe('when there is an existing order, with a matching PPSDK Payment, but without an order token', () => {
```

Comment:
This responsibility has been moved into the test of `src/payment/strategies/ppsdk/ppsdk-payment-resumer.spec.ts`